_drafts/2017-08-24-FedData-release.md

```diff
@@ -0,0 +1,218 @@
+---
+name: FedData-release
+layout: post
+title: FedData - Getting assorted geospatial data into R
+date: 2017-08-24
+authors:
+  - name: Kyle Bocinsky
+    url: https://github.com/bocinsky
+categories:
+  - technotes
+tags:
+  - R
+  - package
+  - data-access
+  - packages
+  - spatial
+  - geospatial
+  - review
+  - onboarding
+  - FedData
+---
+
+The package [FedData](https://github.com/ropensci/FedData) is now part of [rOpenSci](https://ropensci.org/). FedData includes functions to automate downloading geospatial data available from several federated data sources (mainly sources maintained by the US Federal government).
```

Comment:
Rather than "...is now part of [rOpenSci](https://ropensci.org/)" 
you might say "...has gone through software review and is now part of [rOpenSci](https://ropensci.org/packages/)"